kart/dataset_mixins.py

```diff
@@ -55,7 +63,9 @@ def diff_to_working_copy(
         # Subclasses to override.
         pass
 
-    def get_raw_diff_for_subtree(self, other, subtree_name, reverse=False):
+    def get_raw_diff_for_subtree(
+        self, other, subtree_name, reverse=False, data_changes_required=False
```

Comment:
would prefer `data_changes_required=True` I think
- it's the current behaviour
- it's what I would expect by default
- it's consistent with the other defaults that you set everywhere: diff_format=DiffFormat.FULL